app/src/app/export/export.service.ts

```diff
@@ -39,9 +40,12 @@ export class FileService {
   }
 
   public callExportRoute(objectIDs: string, exportType: string, zipping: boolean = false) {
-    (httpFileOptions.params as any).ids = objectIDs;
-    (httpFileOptions.params as any).classname = exportType;
-    (httpFileOptions.params as any).zip = false;
+    const options = httpFileOptions;
```

Comment:
You define the parameters here but they are not set in the call.
Used are httpFileOptions in the callGet. Shouldn´t be:
`    return this.api.callGet<any>(this.servicePrefix + '/object/', options);`

Reply:
Thanks.